src/Tizen.NUI/src/public/BaseComponents/DirectRenderingGLView.cs

```diff
@@ -318,6 +323,23 @@ public void RenderOnce()
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Terminate gl calls forcibly.
+        /// </summary>
+        /// <remarks>
+        /// All API after Terminate don't have any efforts.
```

Comment:
efforts -> effect

Reply:
Fix done! Thank you for the nice pointing